src/modules/delete.js

```diff
@@ -1,8 +1,10 @@
 import createTodo from './createTodo.js';
 
-export default function removeTodo(deleteTask, todosList) {
+function removeTodo(deleteTask, todosList) {
```

Comment:
- [ ] When deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

![Screen Shot 2022-02-17 at 8 55 10](https://user-images.githubusercontent.com/67211919/154509007-71f7e17c-c80b-4d9a-8f7e-d24b2ca53e77.png)


When adding a new item is not supposed to have the same index value and should be updated starting with 1  then 2, 3, 4, etc.

Reply:
Hello @eri8-9 
I really appreciate your feedback. I don't quite understand what and how should I implement it?

Reply:
It would be nice if you'd give a clue on how to do it. 
Thank you.

Reply:
The key is to update the local storage every time you make a change in your code. For example, if you are using one function to update the local storage, every component should call the local storage to be updated inside of the function that deletes the item.

Reply:
@LuisSalas94  Also, you could write a function `orderIds()` that: 

1. Sorts and assigns all the IDs according to the order of the elements of the main tasks array.  

2. Generates a new tasks array according to those results.  The HTML elements will be updated with those results.

Suppose we deleted the second task of the array.  The result would be:
`tasks = [ {task: "first", id: '1'}, {task: "thrid", id: '3'}, {task: "fourth",id: '4'} ]    `

with your `orderIds()` function it would become:
`tasks = [ {task: 'first', id: '1'}, {task: 'third', id: '2'}, {task: 'fourth', id: '3'} ]`



3. Replaces the contents of local storage with the results of said array.

Finally, you will call this function `orderIds()` every time a user deletes tasks.

You can do it 💪

Reply:
@eri8-9 Thanks for the tip you gave me.

Reply:
💪